index.html

```diff
@@ -143,5 +143,27 @@ <h2 class="firsthalf-item firsthalf-title">About me</h2>
       <li class="skills-item skills-tm">Terminal</li>
     </ul>
   </section>
+
+  <section class="contact-us">
+    <h2 class="cont-para">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+    <form class="form" action="https://formspree.io/f/mnqroayv" method="post">
+      <fieldset class="form-fieldset">
+        <input type="text" name="Fullname" class="form-text" placeholder="Full name" maxlength="30" required/>
+        <input type="email" name="Email" class="form-text" placeholder="Email address" required/>
+        <textarea name="Message" maxlength="500" class="form-text form-msg" required>Enter text here</textarea>
+        <input type="submit" id="form-submit" name="submit" value="Get in touch"/>
+      </fieldset>
+    </form>
+    <div class="social-media-icons">
+      <hr/>
+      <ul class="icon-container form-icon-container">
+        <li><a href="" class="icon-element form-icon-element" type="button"><img src="src/github2.png" alt="github"></a></li>
+        <li><a href="" class="icon-element form-icon-element" type="button"><img src="src/i2.png" alt="i"></a></li>
+        <li><a href="" class="icon-element form-icon-element" type="button"><img src="src/v2.png" alt="v"></a></li>
+        <li><a href="" class="icon-element form-icon-element" type="button"><img src="src/t2.png" alt="twitter"></a></li>
+        <li><a href="" class="icon-element form-icon-element" type="button"><img src="src/m2.png" alt="m"></a></li>
+      </ul>
+    </div>
+  </section>
```

Comment:
- [ ]  Please why don't you use the `footer` tag instead of the `section` tag for the contact sections since it is recommended to use in the project requirement and for semantic reasons? :pray: